src/main/java/com/kodilla/ecommercee/service/OrderService.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.ecommercee.service;
+
+import com.kodilla.ecommercee.domain.Order;
+import com.kodilla.ecommercee.dto.OrderDto;
+import com.kodilla.ecommercee.exception.EntityNotFoundException;
+import com.kodilla.ecommercee.mapper.OrderMapper;
+import com.kodilla.ecommercee.repository.OrderRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Service
+public class OrderService {
+
+    private final OrderMapper mapper;
+    private final OrderRepository repository;
+
+    public List<OrderDto> getAll() {
+        return mapper.toDtoList(repository.findAll());
+    }
+
+    public OrderDto getById(long id) throws EntityNotFoundException {
+        return repository.findById(id).map(mapper::toDto).orElseThrow(() -> new EntityNotFoundException(Order.class, id));
+    }
+
+    @Transactional
+    public OrderDto create(final OrderDto orderDto) throws EntityNotFoundException {
+        if(orderDto.getId() != 0){
+            throw new EntityNotFoundException(Order.class, orderDto.getId());
+        }else {
+            return saveAndMapToDto(orderDto);
```

Comment:
klauzula else jest nie potrzebna. Jeżeli warunek if się nie spełni i nie rzuci wyjątkiem to kod jest wykonywany dalej, a więc po if możesz od razu dać return (...)
spójrz na mój kod i go przeanalizuj to będziesz wiedział o co chodzi :)